patches/minecraft/net/minecraft/client/gui/GuiBossOverlay.java.patch

```diff
@@ -0,0 +1,19 @@
+--- ../src-base/minecraft/net/minecraft/client/gui/GuiBossOverlay.java
++++ ../src-work/minecraft/net/minecraft/client/gui/GuiBossOverlay.java
+@@ -35,13 +35,16 @@
+             for (BossInfoLerping bossinfolerping : this.field_184060_g.values())
+             {
+                 int k = i / 2 - 91;
++                if (!net.minecraftforge.client.ForgeHooksClient.bossBarRenderPre(scaledresolution, bossinfolerping, k, j)) {
+                 GlStateManager.func_179131_c(1.0F, 1.0F, 1.0F, 1.0F);
+                 this.field_184059_f.func_110434_K().func_110577_a(field_184058_a);
+                 this.func_184052_a(k, j, bossinfolerping);
+                 String s = bossinfolerping.func_186744_e().func_150254_d();
+                 this.field_184059_f.field_71466_p.func_175063_a(s, (float)(i / 2 - this.field_184059_f.field_71466_p.func_78256_a(s) / 2), (float)(j - 9), 16777215);
++                }
+                 j += 10 + this.field_184059_f.field_71466_p.field_78288_b;
+ 
```

Comment:
This may be an issue, probabbly should figure out how to ask the event how tall the new render is?